python/ml4ir/base/model/relevance_model.py

```diff
@@ -602,14 +603,16 @@ def save(
         pad_sequence: bool, optional
             Value defining if sequences should be padded for SequenceExample proto inputs at serving time.
             Set this to False if you want to not handle padded scores.
+        sub_dir: str, optional
+            sub directory name to save the model into
 
         Notes
         -----
         All the functions passed under `preprocessing_keys_to_fns` here must be
         serializable tensor graph operations
         """
 
-        model_file = os.path.join(models_dir, "final")
+        model_file = os.path.join(models_dir, sub_dir)
```

Comment:
add the flexibility to save any model (original model, calibrated model, ...)

Reply:
Does this break any of the pipelines we already have? Since you are adding this to the `base` class it will propagate to all models.

Reply:
It should not break as I have set the default value to `'final'`. See line 575:
`sub_dir: str = 'final'`